Makefile

```diff
@@ -483,6 +540,13 @@ run-cc-tests: all-cc-tests
 	@echo
 	python3 l3_dump.py $(L3_CC_SMALL_DATA) ./$(CC_UNIT_TEST_BIN)
 
+run-unit-tests: all-unit-tests
+	@echo
+	@echo '---- Run L3 unit-tests: ----'
+	./$(L3_C_UNIT_TEST_BIN)
+	@echo
+	python3 l3_dump.py $(L3_C_UNIT_TEST_DATA) ./$(L3_C_UNIT_TEST_BIN)
```

Comment:
For now, we simply execute the dump script which will only print 3 log-entries. 

I tried to "extend" this script to add code to trip the error, but found all that of not too much value. For now, just live with an incomplete execution.